### ISSUE TITLE
ScholarSphere deposit not showing up in RMD or Unpaywall

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-libraries/researcher-metadata#552]
I’m looking at an [RMD pub](https://metadata.libraries.psu.edu/admin/publication/283343) and a [ScholarSphere deposit](https://scholarsphere.psu.edu/resources/b614b8c9-ac37-4c15-9d05-e6779f2c7dc0) — can’t figure out why the ScholarSphere importer is not picking it up (and neither is Unpaywall). 

The ScholarSphere deposit has two DOIs associated -- one is the correct one from the publisher. The other is a ScholarSphere DOI that the depositor minted (which really should not happen for postprint deposits). This may be the issue.

**Comment by ajkiessl** [comment]
Yeah the two different DOIs is causing the issue.  The proper DOI is actually associated with the work versions for this work, and the minted DOI is associated with the work.  The minted work DOI seems to be given preference in the API.  We may be able to tweak how ScholarSphere is handling multiple DOIs, but I'd have to take some time to dig a little deeper in the code.

**Comment by anaelizabethenriquez** [comment]
@ajkiessl Should I add an issue about this in the ScholarSphere backlog, or is it better to just leave this ticket here?

Also, any preference about how I fix this in the short term? I'm thinking I'll just add a link in RMD with ScholarSphere as the source.

**Comment by ajkiessl** [comment]
@anaelizabethenriquez Yeah go ahead and add an issue to ScholarSphere for this.  I was thinking an admin could just delete the minted DOI in ScholarSphere, but maybe that's not what we want to do?  If that's not an option then adding the link yourself in RMD should work.
